app/src/NativeBase/Components/Forms/DatePicker.tsx

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file DatePicker.
+ */
+
+import React, { FC, useState } from 'react'
+import CalendarPicker from 'react-native-calendar-picker'
+import { Pressable, Dimensions } from 'react-native'
+import { VStack, Box, Text, Input, useColorMode, Icon } from 'native-base'
+import dayjs from 'dayjs'
+import StaTheme from '../../Theme/StaTheme'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+interface DatePickerProps {
+  description: string
+  maximumDate?: Date
+  minimumDate?: Date
+  onChange: (date: Date) => void
+  value: Date
+}
+
+const DatePicker: FC<DatePickerProps> = ({
+  description,
+  maximumDate,
+  minimumDate,
+  onChange,
+  value,
+}) => {
+  const [pickerActive, setPickerActive] = useState(false)
+  const [selectedStartDate, setSelectedStartDate] = useState(null)
+  const [todaySelected, setTodaySelected] = useState(false)
+  const { colorMode } = useColorMode()
+  const { width } = Dimensions.get('window')
+
+  const handleChange = (value: Date) => {
+    onChange(value)
+    setSelectedStartDate(value)
+    setPickerActive(false)
+    if (dayjs(value).format('DD/MM/YY') === dayjs().format('DD/MM/YY')) {
```

Comment:
Tiny thing but there's a built-in dayjs method [isSame](https://day.js.org/docs/en/query/is-same) you could use here -- e.g. if you want to check `value` has the same day as today: `if (dayjs(value).isSame(dayjs(), 'day')) {`